src/main/java/frc/robot/commands/AutoAlignMethods.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.AlignPosition;
+import frc.robot.commands.FlyWheelCommand;
+
+public class AutoAlignMethods extends Command{
```

Comment:
Might consider renaming this to something like AlignmentTransitions, since we'll probably want to use this for teleop stuff and including "Methods" in the name is a bit redundant.

Reply:
done